bootstrap_cfn/stacks/ec2.json

```diff
@@ -8,11 +8,26 @@
         "SecurityGroupIngress": []
     }
   },
+  "SaltSG": {
+    "Type": "AWS::EC2::SecurityGroup",
+    "Properties": {
+        "VpcId" : { "Ref" : "VPC" },
+        "GroupDescription": "Salt Comms Security Group",
+        "SecurityGroupIngress": [
+                                    {
+                                        "IpProtocol": "tcp",
+                                        "FromPort": 4505,
+                                        "ToPort": 4506,
```

Comment:
Good spot - I think this should be fromport: 4505, toport:4505 - from being 0.0.0.0/0:4505 to somewhere in the VPC on port 4505